launch/pipeline/service.py

```diff
@@ -45,10 +71,26 @@ def _init_callable_service(self):
             ), "`kwargs` is given, but the service is not a class"
             self._callable_service = self.service
 
+    def set_make_request(
```

Comment:
In https://github.com/scaleapi/models/pull/3187/files it looks like the attribute being looked for is `register_make_request` as opposed to `set_make_request`, is that the intended behavior?